### DISCUSSION TITLE
Birt image inside dynamic text boxe

### DISCUSSION THREAD

**Discussion by Jonasbr93** [eclipse-birt/birt#829] [Q&A] [answered]
Hello ! im new with Eclipse and also birt and im developing a project for lms - sap.

What i need to do is basically inset a image inside a dynamic text box if someone could help me with that it would great.

Also i dont know if its possible to insert the same image if the dynamic text box has more than one line.

its basically and arrow.... what ive tryed to do so far was to insert that image of the arrow inside Embedded Images and write inside the dynamic text box somenthing like this :

``<IMAGE` name="seta.png" source="embedded"></IMAGE>'> + row._outer["FORM"]`

As you pros may know it dint work.
Let me know if you can help me in any of teh case i would be very happy if at least i could have just the image inside of the box. Many thanks in advance
![Screenshot 2022-02-10 235553](https://user-images.githubusercontent.com/71432032/153565909-c0cad187-046b-47be-9f86-9de50fe71745.png)
.

**Comment by hvbtup** [comment]
The dynamic text items support only a *very* restricted subset of HTML. Basically, that means character formatting.
Images inside HTML are not supported AFAIK (even if they may work).

What you can do instead is to use a unicode character, e.g. "\u25b6". You can put this into a green `<span>` element. 

If you want to use it for a bullet list, you should consider using a separate column (of a grid or table item), then it will look nicer if the FORM text is long and needs line-breaking.

**Comment by hvbtup** [comment, marked as answer]
![grafik](https://user-images.githubusercontent.com/7644776/153579135-d57e5aba-3f73-4cc8-9984-8e755414f500.png)

Works for me - note that I used 14pt for the normal text and 16pt for the triangle bullet. It looks like this:
![grafik](https://user-images.githubusercontent.com/7644776/153579594-96387999-8b4e-4a15-ac0d-3c641a2a7e55.png)

**Reply by Jonasbr93** [reply]
Never mind ! i deleted the post that i did before but was not on time ahah sorry for that! its working :) thanks! i will try to understand later what u meant for the bullet list, but for now its good like that! once again thank you!
